x-pack/plugins/monitoring/public/plugin.ts

```diff
@@ -93,6 +93,7 @@ export class MonitoringPlugin
       category: DEFAULT_APP_CATEGORIES.management,
       mount: async (params: AppMountParameters) => {
         const [coreStart, pluginsStart] = await core.getStartServices();
+        await pluginsStart.kibanaLegacy.loadAngularBootstrap();
```

Comment:
I didn't measure it, but I guess that this could make loading the stack monitoring page slightly slower. What do you think if we do something like this:
```suggestion
        const [, { AngularApp }] = await Promise.all([
          pluginsStart.kibanaLegacy.loadAngularBootstrap(),
          import('./angular'),
        ]);
```
Does it make sense?

Reply:
Sure, I will commit your suggestion :)